scripts/bytecodecompare/prepare_report.py

```diff
@@ -224,6 +230,10 @@ def prepare_compiler_input(
 
     settings = CompilerSettings.from_preset(preset)
 
+    command_line = []
+    if execution_arch == ExecutionArchitecture.X86_64.value:
+        command_line = ["/usr/bin/arch", "-64", "-x86_64"]
```

Comment:
You should assert that it must be one or the other value, because if we add a new one, this code will have to be updated:
```python
else:
    assert execution_arch == ExecutionArchitecture.NATIVE.value
```